test/io/serializers/deserialize-value.chpl

```diff
@@ -14,7 +14,7 @@ record R {
     writeln("IN R.init");
     ref des = deserializer;
     des.startRecord(reader, "R", 1);
-    this.x = des.deserializeField(reader, "x", int);
+    this.x = des.readField(reader, "x", int);
```

Comment:
Doesn't `des` need to be the value returned by `startRecord` now?

Reply:
Yes, it does. This snippet didn't result in an error because it's in the initializer, which this test shouldn't be invoking at all, which means the compiler won't attempt to resolve it either. I'll fix it regardless.